circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainS3S3IntegrationTest/unpartitioned-single-table-full-overwrite-replication.yml

```diff
@@ -0,0 +1,10 @@
+table-replications:
```

Comment:
Having looked through the above I'm not sure we need to add these tests to all combinations of HDFS and S3 as they probably don't really test anything new in each of them right? Perhaps create a new `CircusTrainReplicationModeIntegrationTest` and then put the two tests into that and have it use S3 to S3 for the location on each side (as that's becoming the most used file system on both sides now). 

At some point we should review the other integration tests and see if there is other overlap that isn't needed that can be refactored out to reduce the number of these.

Reply:
Yeah it felt pretty repetitive, this sounds like a better idea 👍